frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/ToggleListingCommitsAction.java

```diff
@@ -45,16 +44,6 @@ public void onUpdate(AnActionEvent anActionEvent) {
       return;
     }
 
-    boolean noChildBranchExists = branchLayout.get().getRootEntries()
-        .exists(rootBranch -> rootBranch.getChildren().isEmpty());
```

Comment:
There was a bug here... it should've been `.forAll`, not `.exists`

Reply:
But anyway, we don't need this check, `anyCommitExists == false` covers the case of no child branch as well